[2024-01-01T22:11:23.080Z] Error in the main function
Error: Unsupported operation: [object Object]
[2024-01-01T22:13:17.613Z] Error in the main function
Error: Unsupported operation: newInvoice
[2024-01-01T22:14:19.880Z] Error in the main function
TypeError: Cannot convert undefined or null to object
[2024-01-01T22:15:58.158Z] Error in the main function
TypeError: Cannot convert undefined or null to object
[2024-01-01T22:16:15.572Z] Error in the main function
TypeError: Cannot convert undefined or null to object
[2024-01-01T22:16:24.753Z] Error in the main function
TypeError: Cannot convert undefined or null to object
[2024-01-01T22:24:20.193Z] Error in the main function
TypeError: Cannot convert undefined or null to object
[2024-01-01T22:25:19.707Z] Error in the main function
TypeError: Cannot convert undefined or null to object
[2024-01-01T22:25:30.170Z] Error in the main function
TypeError: Cannot convert undefined or null to object
[2024-01-01T22:26:26.673Z] Error in the main function
Error: Invalid or missing wholeInvoice in data
[2024-01-03T00:47:07.944Z] Error getting access token
Error: request to https://www.googleapis.com/oauth2/v4/token failed, reason: socket hang up


[2024-01-03T00:47:07.951Z] Error in getLastDocument
Error getting access token
Error: request to https://www.googleapis.com/oauth2/v4/token failed, reason: socket hang up


[2024-01-03T00:47:07.953Z] Error in the main function
Error in getLastDocument
Error getting access token
Error: request to https://www.googleapis.com/oauth2/v4/token failed, reason: socket hang up
{}

[2024-01-09T01:58:57.911Z] Error in interactWithFirestore
Error: Unsupported operation: writeData
{
  "invoiceid": null,
  "completedDate": "todays date",
  "fullInvoice": {
    "A": "any string value",
    "B": "any string value",
    "C": "any string value",
    "D": "any string value",
    "invoicemap": {
      "key1": "value1",
      "key2": "value2"
    }
  }
}

[2024-01-09T01:59:35.642Z] Error in interactWithFirestore
[object Object]
{
  "invoiceid": null,
  "completedDate": "todays date",
  "fullInvoice": {
    "A": "any string value",
    "B": "any string value",
    "C": "any string value",
    "D": "any string value",
    "invoicemap": {
      "key1": "value1",
      "key2": "value2"
    }
  }
}

[2024-01-09T02:11:20.369Z] Error in interactWithFirestore
Error: Unsupported operation: writeData
{
  "invoiceid": null,
  "completedDate": "todays date",
  "fullInvoice": {
    "A": "any string value",
    "B": "any string value",
    "C": "any string value",
    "D": "any string value",
    "invoicemap": {
      "key1": "value1",
      "key2": "value2"
    }
  }
}

[2024-01-09T02:20:19.045Z] Error in interactWithFirestore
[object Object]
{
  "documentId": "1",
  "updateFields": {
    "A": "any thing",
    "B": "any string value",
    "C": "any string value",
    "D": "any string value",
    "invoiceDetails": {
      "key1": "value1",
      "key2": "value2",
      "key3": "value3",
      "key4": "value4",
      "key5": "value5",
      "key6": "value6"
    }
  }
}

[2024-01-09T02:23:19.383Z] Error in updateData
[object Object]
{
  "documentId": "1",
  "updateFields": {
    "A": "any thing",
    "B": "any string value",
    "C": "any string value",
    "D": "any string value",
    "invoiceDetails": {
      "key1": "value1",
      "key2": "value2",
      "key3": "value3",
      "key4": "value4",
      "key5": "value5",
      "key6": "value6"
    }
  }
}

[2024-01-09T02:26:03.996Z] Error in updateData
Error Details: Request failed with status code 400
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "documentId": "1",
  "updateFields": {
    "A": "any thing",
    "B": "any string value",
    "C": "any string value",
    "D": "any string value",
    "invoiceDetails": {
      "key1": "value1",
      "key2": "value2",
      "key3": "value3",
      "key4": "value4",
      "key5": "value5",
      "key6": "value6"
    }
  }
}

[2024-01-09T03:11:18.642Z] Error in interactWithFirestore
Error Details: ENOENT: no such file or directory, open 'C:\Users\theus\yeaa\invoicedashboardfcr-b62134e84d26.json'
Stack Trace: Error: ENOENT: no such file or directory, open 'C:\Users\theus\yeaa\invoicedashboardfcr-b62134e84d26.json'
{
  "A": "any strineeeg value",
  "B": "any string value",
  "C": "any string value",
  "D": "any string value",
  "invoiceDetails": {
    "key1": "value1",
    "key2": "value2",
    "key3": "value3",
    "key4": "value4",
    "key5": "value5",
    "key6": "value6"
  }
}

[2024-01-09T03:30:10.011Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/pbqNZyh2tDSxRfVscM7q\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"pbqNZyh2tDSxRfVscM7q\"\n}"

[2024-01-09T03:38:38.804Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/MDkWGO117orHDg49bEVw\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"MDkWGO117orHDg49bEVw\"\n}"

[2024-01-09T03:43:18.633Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/niQ0Cm0drrFDLnPyIvCE\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"niQ0Cm0drrFDLnPyIvCE\"\n}"

[2024-01-09T03:48:50.907Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/Ra8jzwitset8cr0MPPLp\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"Ra8jzwitset8cr0MPPLp\"\n}"

[2024-01-09T03:49:23.206Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/Zi7PAgboGccAMCX32gmn\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"Zi7PAgboGccAMCX32gmn\"\n}"

[2024-01-09T03:53:40.147Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/jQFNdpm4L6eHJNoGhuQK\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"jQFNdpm4L6eHJNoGhuQK\"\n}"

[2024-01-09T03:54:27.912Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/Hs3c3VDKI6JwLvIiWuIP\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"Hs3c3VDKI6JwLvIiWuIP\"\n}"

[2024-01-09T03:55:47.013Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/XNa6PFjHgJwnOWPqXJ3O\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"XNa6PFjHgJwnOWPqXJ3O\"\n}"

[2024-01-09T03:56:06.270Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/T5JAS7Z59Fa57VJXroPl\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"T5JAS7Z59Fa57VJXroPl\"\n}"

[2024-01-09T03:57:28.302Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/PLjiMSxxufskA7ExoP1n\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"PLjiMSxxufskA7ExoP1n\"\n}"

[2024-01-09T03:58:37.125Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/H2o1T5jBHzFjOBJPXugb\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"H2o1T5jBHzFjOBJPXugb\"\n}"

[2024-01-09T04:00:00.110Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/wEbltSiKVWqyBt6oCp7i\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"wEbltSiKVWqyBt6oCp7i\"\n}"

[2024-01-09T04:02:11.332Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/7glU8WcjaEiL2iVEKX4x\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"7glU8WcjaEiL2iVEKX4x\"\n}"

[2024-01-09T04:04:30.566Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/xgpdNWqR0EXfnbU8gPAS\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"xgpdNWqR0EXfnbU8gPAS\"\n}"

[2024-01-09T04:05:29.448Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/WI7TADTKPDFROCaUhYBz\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"WI7TADTKPDFROCaUhYBz\"\n}"

[2024-01-09T04:05:44.498Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/NC3Pcy2iElgrwNRJQECq\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"NC3Pcy2iElgrwNRJQECq\"\n}"

[2024-01-09T04:08:02.555Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/VSH1AUmN185UKWTv90LO\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"VSH1AUmN185UKWTv90LO\"\n}"

[2024-01-09T04:08:45.145Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/h2omTSNSp1acXNHunbjR\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"h2omTSNSp1acXNHunbjR\"\n}"

[2024-01-09T04:09:34.746Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/8sVfW7DZR9gOJ29gIBE8\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"8sVfW7DZR9gOJ29gIBE8\"\n}"

[2024-01-09T04:10:11.145Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/TxM3iDD36AU6skKPdLXq\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"TxM3iDD36AU6skKPdLXq\"\n}"

[2024-01-09T04:12:25.624Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/jJJUYQg2hHX4zxbfJN9l\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"jJJUYQg2hHX4zxbfJN9l\"\n}"

[2024-01-09T04:15:09.776Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/xEysXrwp9NERDeNUjAkA\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"xEysXrwp9NERDeNUjAkA\"\n}"

[2024-01-09T04:15:12.708Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/kc3ETK5PK0U2etRQplm6\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"kc3ETK5PK0U2etRQplm6\"\n}"

[2024-01-09T04:15:16.797Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/rHKaUsId4TPnaE8JeOJB\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"rHKaUsId4TPnaE8JeOJB\"\n}"

[2024-01-09T04:24:21.548Z] Error in updateData
Error Details: Cannot convert undefined or null to object
Stack Trace: TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at interactWithFirestore (C:\Users\theus\yeaa\FCRdashboard\server\models\firestoreApp.js:147:196)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async C:\Users\theus\yeaa\FCRdashboard\server\routes.js:43:22
"{\n  \"A\": \"123\",\n  \"B\": \"01/07/2024\",\n  \"C\": \"H&E\",\n  \"D\": \"Kevin street\",\n  \"creationDate\": \"2024-01-09T04:24:21.252Z\",\n  \"invoiceDetails\": {\n    \"1A\": \"xyxy123123\",\n    \"1B\": \"new shoes\",\n    \"1C\": \"150\",\n    \"2A\": \"Some\",\n    \"2B\": \"thing\",\n    \"2C\": \"1.2\",\n    \"3A\": \"\",\n    \"3B\": \"\",\n    \"3C\": \"\",\n    \"4A\": \"\",\n    \"4B\": \"\",\n    \"4C\": \"\",\n    \"5A\": \"\",\n    \"5B\": \"\",\n    \"5C\": \"\",\n    \"6A\": \"\",\n    \"6B\": \"\",\n    \"6C\": \"\",\n    \"7A\": \"\",\n    \"7B\": \"\",\n    \"7C\": \"\",\n    \"8A\": \"\",\n    \"8B\": \"\",\n    \"8C\": \"\",\n    \"9A\": \"\",\n    \"9B\": \"\",\n    \"9C\": \"\",\n    \"10A\": \"\",\n    \"10B\": \"\",\n    \"10C\": \"\",\n    \"11A\": \"\",\n    \"11B\": \"\",\n    \"11C\": \"\",\n    \"12A\": \"\",\n    \"12B\": \"\",\n    \"12C\": \"\",\n    \"13A\": \"\",\n    \"13B\": \"\",\n    \"13C\": \"\",\n    \"14A\": \"\",\n    \"14B\": \"\",\n    \"14C\": \"\",\n    \"15A\": \"\",\n    \"15B\": \"\",\n    \"15C\": \"\",\n    \"16A\": \"\",\n    \"16B\": \"\",\n    \"16C\": \"\",\n    \"17A\": \"\",\n    \"17B\": \"\",\n    \"17C\": \"\",\n    \"18A\": \"\",\n    \"18B\": \"\",\n    \"18C\": \"\",\n    \"19A\": \"\",\n    \"19B\": \"\",\n    \"19C\": \"\",\n    \"20A\": \"\",\n    \"20B\": \"\",\n    \"20C\": \"\",\n    \"21A\": \"\",\n    \"21B\": \"\",\n    \"21C\": \"\",\n    \"22A\": \"\",\n    \"22B\": \"\",\n    \"22C\": \"\",\n    \"23A\": \"\",\n    \"23B\": \"\",\n    \"23C\": \"\",\n    \"24A\": \"\",\n    \"24B\": \"\",\n    \"24C\": \"\",\n    \"25A\": \"\",\n    \"25B\": \"\",\n    \"25C\": \"\",\n    \"26A\": \"\",\n    \"26B\": \"\",\n    \"26C\": \"\"\n  }\n}"

[2024-01-09T04:30:44.802Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/WN0qJaevuBZCWodqTOFi\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"WN0qJaevuBZCWodqTOFi\"\n}"

[2024-01-15T00:59:08.079Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/undefined\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{}"

[2024-01-15T00:59:34.205Z] Error in readData
Error Details: Request failed with status code 404
Response Data: {
  "error": {
    "code": 404,
    "message": "Document \"projects/invoicedashboardfcr/databases/invoicedatastore/documents/invoices/undefined\" not found.",
    "status": "NOT_FOUND"
  }
}
Stack Trace: AxiosError: Request failed with status code 404
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{}"

[2024-01-23T22:47:40.877Z] Error in interactWithFirestore
Error Details: request to https://www.googleapis.com/oauth2/v4/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
Stack Trace: Error: request to https://www.googleapis.com/oauth2/v4/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
    at Gaxios._request (C:\Users\theus\yeaa\FCRdashboard\node_modules\gaxios\build\src\gaxios.js:148:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async GoogleToken.requestToken (C:\Users\theus\yeaa\FCRdashboard\node_modules\gtoken\build\src\index.js:230:23)
    at async GoogleToken.getTokenAsync (C:\Users\theus\yeaa\FCRdashboard\node_modules\gtoken\build\src\index.js:137:20)
    at async JWT.refreshTokenNoCache (C:\Users\theus\yeaa\FCRdashboard\node_modules\google-auth-library\build\src\auth\jwtclient.js:165:23)
    at async JWT.refreshAccessTokenAsync (C:\Users\theus\yeaa\FCRdashboard\node_modules\google-auth-library\build\src\auth\oauth2client.js:209:19)
    at async JWT.getAccessTokenAsync (C:\Users\theus\yeaa\FCRdashboard\node_modules\google-auth-library\build\src\auth\oauth2client.js:238:23)
    at async interactWithFirestore (C:\Users\theus\yeaa\FCRdashboard\server\models\firestoreApp.js:111:30)
    at async C:\Users\theus\yeaa\FCRdashboard\server\routes.js:70:31
100

[2024-01-23T23:32:22.390Z] Error in interactWithFirestore
Error Details: Invalid time value
Stack Trace: RangeError: Invalid time value
    at Date.toISOString (<anonymous>)
    at convertToISO (C:\Users\theus\yeaa\FCRdashboard\server\models\firestoreApp.js:49:48)
    at interactWithFirestore (C:\Users\theus\yeaa\FCRdashboard\server\models\firestoreApp.js:214:54)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async C:\Users\theus\yeaa\FCRdashboard\server\routes.js:93:24
{
  "numberOfResults": "ALL",
  "fromAccount": "ALL",
  "startDate": "2023-12-01",
  "endDate": "2024-01-01"
}

[2024-01-23T23:32:26.024Z] Error in interactWithFirestore
Error Details: Invalid time value
Stack Trace: RangeError: Invalid time value
    at Date.toISOString (<anonymous>)
    at convertToISO (C:\Users\theus\yeaa\FCRdashboard\server\models\firestoreApp.js:49:48)
    at interactWithFirestore (C:\Users\theus\yeaa\FCRdashboard\server\models\firestoreApp.js:214:54)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async C:\Users\theus\yeaa\FCRdashboard\server\routes.js:93:24
{
  "numberOfResults": "ALL",
  "fromAccount": "ALL",
  "startDate": "2023-12-01",
  "endDate": "2024-01-01"
}

[2024-01-23T23:32:35.747Z] Error in interactWithFirestore
Error Details: Invalid time value
Stack Trace: RangeError: Invalid time value
    at Date.toISOString (<anonymous>)
    at convertToISO (C:\Users\theus\yeaa\FCRdashboard\server\models\firestoreApp.js:49:48)
    at interactWithFirestore (C:\Users\theus\yeaa\FCRdashboard\server\models\firestoreApp.js:214:54)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async C:\Users\theus\yeaa\FCRdashboard\server\routes.js:93:24
{
  "numberOfResults": "ALL",
  "fromAccount": "ALL",
  "startDate": "2023-12-01",
  "endDate": "2024-01-01"
}

[2024-01-23T23:34:42.495Z] Error in interactWithFirestore
Error Details: Invalid time value
Stack Trace: RangeError: Invalid time value
    at Date.toISOString (<anonymous>)
    at convertToISO (C:\Users\theus\yeaa\FCRdashboard\server\models\firestoreApp.js:49:48)
    at interactWithFirestore (C:\Users\theus\yeaa\FCRdashboard\server\models\firestoreApp.js:214:54)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async C:\Users\theus\yeaa\FCRdashboard\server\routes.js:93:24
{
  "numberOfResults": "ALL",
  "fromAccount": "ALL",
  "startDate": "2023-12-01",
  "endDate": "2024-01-01"
}

[2024-01-24T01:46:37.159Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [
  {
    "error": {
      "code": 400,
      "message": "The query requires an index. You can create it here: https://console.firebase.google.com/v1/r/project/invoicedashboardfcr/firestore/databases/invoicedatastore/indexes?create_composite=Cltwcm9qZWN0cy9pbnZvaWNlZGFzaGJvYXJkZmNyL2RhdGFiYXNlcy9pbnZvaWNlZGF0YXN0b3JlL2NvbGxlY3Rpb25Hcm91cHMvaW52b2ljZXMvaW5kZXhlcy9fEAEaBQoBQxABGhAKDGNyZWF0aW9uRGF0ZRACGgwKCF9fbmFtZV9fEAI",
      "status": "FAILED_PRECONDITION"
    }
  }
]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"numberOfResults\": \"5\",\n  \"fromAccount\": \"test\",\n  \"startDate\": \"\",\n  \"endDate\": \"\"\n}"

[2024-01-26T22:13:08.799Z] Error in updateData
Error Details: Request failed with status code 400
Response Data: {
  "error": {
    "code": 400,
    "message": "Invalid property path \"1A\". Unquoted property paths must match regex ([a-zA-Z_][a-zA-Z_0-9]*), and quoted property paths must match regex (`(?:[^`\\\\]|(?:\\\\.))+`)",
    "status": "INVALID_ARGUMENT"
  }
}
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"4mNCzgCE9PYYGS7Z7Yea\",\n  \"updateFields\": {\n    \"A\": \"012423\",\n    \"B\": \"01/24/2024\",\n    \"C\": \"test2\",\n    \"D\": \"test\",\n    \"E\": \"test\",\n    \"1A\": \"\",\n    \"1B\": \"\",\n    \"1C\": \"\",\n    \"2A\": \"\",\n    \"2B\": \"\",\n    \"2C\": \"991\",\n    \"3A\": \"\",\n    \"3B\": \"eeeeee\",\n    \"3C\": \"\",\n    \"4A\": \"\",\n    \"4B\": \"\",\n    \"4C\": \"\",\n    \"5A\": \"\",\n    \"5B\": \"\",\n    \"5C\": \"\",\n    \"6A\": \"\",\n    \"6B\": \"\",\n    \"6C\": \"\",\n    \"7A\": \"\",\n    \"7B\": \"\",\n    \"7C\": \"\",\n    \"8A\": \"\",\n    \"8B\": \"\",\n    \"8C\": \"\",\n    \"9A\": \"\",\n    \"9B\": \"\",\n    \"9C\": \"\",\n    \"10A\": \"\",\n    \"10B\": \"\",\n    \"10C\": \"\",\n    \"11A\": \"\",\n    \"11B\": \"\",\n    \"11C\": \"\",\n    \"12A\": \"\",\n    \"12B\": \"\",\n    \"12C\": \"\",\n    \"13A\": \"\",\n    \"13B\": \"\",\n    \"13C\": \"\",\n    \"14A\": \"\",\n    \"14B\": \"\",\n    \"14C\": \"\",\n    \"15A\": \"\",\n    \"15B\": \"\",\n    \"15C\": \"\",\n    \"16A\": \"\",\n    \"16B\": \"\",\n    \"16C\": \"\",\n    \"17A\": \"\",\n    \"17B\": \"\",\n    \"17C\": \"\",\n    \"18A\": \"\",\n    \"18B\": \"\",\n    \"18C\": \"\",\n    \"19A\": \"\",\n    \"19B\": \"\",\n    \"19C\": \"\",\n    \"20A\": \"\",\n    \"20B\": \"\",\n    \"20C\": \"\",\n    \"21A\": \"\",\n    \"21B\": \"\",\n    \"21C\": \"\",\n    \"22A\": \"\",\n    \"22B\": \"\",\n    \"22C\": \"\",\n    \"23A\": \"\",\n    \"23B\": \"\",\n    \"23C\": \"\",\n    \"24A\": \"\",\n    \"24B\": \"\",\n    \"24C\": \"\",\n    \"25A\": \"\",\n    \"25B\": \"\",\n    \"25C\": \"\",\n    \"26A\": \"\",\n    \"26B\": \"\",\n    \"26C\": \"\",\n    \"27A\": \"\",\n    \"27B\": \"\",\n    \"27C\": \"\",\n    \"docID\": \"4mNCzgCE9PYYGS7Z7Yea\"\n  }\n}"

[2024-01-26T22:29:55.395Z] Error in updateData
Error Details: updateResponse is not defined
Stack Trace: ReferenceError: updateResponse is not defined
    at interactWithFirestore (C:\Users\theus\yeaa\FCRdashboard\server\models\firestoreApp.js:155:62)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async C:\Users\theus\yeaa\FCRdashboard\server\routes.js:115:24
"{\n  \"documentId\": \"4mNCzgCE9PYYGS7Z7Yea\",\n  \"updateFields\": {\n    \"A\": \"012423\",\n    \"B\": \"01/24/2024\",\n    \"C\": \"teste\",\n    \"D\": \"test\",\n    \"E\": \"test\",\n    \"creationDate\": \"2024-01-26T22:29:55.284Z\",\n    \"invoiceDetails\": {\n      \"1A\": \"\",\n      \"1B\": \"\",\n      \"1C\": \"\",\n      \"2A\": \"\",\n      \"2B\": \"\",\n      \"2C\": \"\",\n      \"3A\": \"\",\n      \"3B\": \"\",\n      \"3C\": \"\",\n      \"4A\": \"\",\n      \"4B\": \"\",\n      \"4C\": \"\",\n      \"5A\": \"\",\n      \"5B\": \"\",\n      \"5C\": \"\",\n      \"6A\": \"\",\n      \"6B\": \"\",\n      \"6C\": \"\",\n      \"7A\": \"\",\n      \"7B\": \"\",\n      \"7C\": \"\",\n      \"8A\": \"\",\n      \"8B\": \"\",\n      \"8C\": \"\",\n      \"9A\": \"\",\n      \"9B\": \"\",\n      \"9C\": \"\",\n      \"10A\": \"\",\n      \"10B\": \"\",\n      \"10C\": \"\",\n      \"11A\": \"\",\n      \"11B\": \"\",\n      \"11C\": \"\",\n      \"12A\": \"\",\n      \"12B\": \"\",\n      \"12C\": \"\",\n      \"13A\": \"\",\n      \"13B\": \"\",\n      \"13C\": \"\",\n      \"14A\": \"\",\n      \"14B\": \"\",\n      \"14C\": \"\",\n      \"15A\": \"\",\n      \"15B\": \"\",\n      \"15C\": \"\",\n      \"16A\": \"\",\n      \"16B\": \"\",\n      \"16C\": \"\",\n      \"17A\": \"\",\n      \"17B\": \"\",\n      \"17C\": \"\",\n      \"18A\": \"\",\n      \"18B\": \"\",\n      \"18C\": \"\",\n      \"19A\": \"\",\n      \"19B\": \"\",\n      \"19C\": \"\",\n      \"20A\": \"\",\n      \"20B\": \"\",\n      \"20C\": \"\",\n      \"21A\": \"\",\n      \"21B\": \"\",\n      \"21C\": \"\",\n      \"22A\": \"\",\n      \"22B\": \"\",\n      \"22C\": \"\",\n      \"23A\": \"\",\n      \"23B\": \"\",\n      \"23C\": \"\",\n      \"24A\": \"\",\n      \"24B\": \"\",\n      \"24C\": \"\",\n      \"25A\": \"\",\n      \"25B\": \"\",\n      \"25C\": \"\",\n      \"26A\": \"\",\n      \"26B\": \"\",\n      \"26C\": \"\",\n      \"27A\": \"\",\n      \"27B\": \"\",\n      \"27C\": \"\"\n    }\n  }\n}"

[2024-01-26T23:10:17.738Z] Error in updateData
Error Details: Request failed with status code 400
Response Data: {
  "error": {
    "code": 400,
    "message": "Invalid JSON payload received. Unknown name \"updateMask\" at 'document': Cannot find field.",
    "status": "INVALID_ARGUMENT",
    "details": [
      {
        "@type": "type.googleapis.com/google.rpc.BadRequest",
        "fieldViolations": [
          {
            "field": "document",
            "description": "Invalid JSON payload received. Unknown name \"updateMask\" at 'document': Cannot find field."
          }
        ]
      }
    ]
  }
}
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\n  \"documentId\": \"9gSUTIV3w3SIBRrKpdPa\",\n  \"updateFields\": {\n    \"A\": \"123324\",\n    \"B\": \"01/08/2024\",\n    \"C\": \"tesssseee\",\n    \"D\": \"eesxaz\",\n    \"E\": \"kjfnhebsdhnajm\",\n    \"creationDate\": \"2024-01-26T23:10:17.520Z\",\n    \"invoiceDetails\": {\n      \"1A\": \"\",\n      \"1B\": \"\",\n      \"1C\": \"686868\",\n      \"2A\": \"\",\n      \"2B\": \"\",\n      \"2C\": \"\",\n      \"3A\": \"\",\n      \"3B\": \"\",\n      \"3C\": \"1111\",\n      \"4A\": \"\",\n      \"4B\": \"\",\n      \"4C\": \"\",\n      \"5A\": \"\",\n      \"5B\": \"\",\n      \"5C\": \"\",\n      \"6A\": \"\",\n      \"6B\": \"\",\n      \"6C\": \"\",\n      \"7A\": \"\",\n      \"7B\": \"\",\n      \"7C\": \"\",\n      \"8A\": \"\",\n      \"8B\": \"\",\n      \"8C\": \"\",\n      \"9A\": \"\",\n      \"9B\": \"\",\n      \"9C\": \"\",\n      \"10A\": \"\",\n      \"10B\": \"\",\n      \"10C\": \"\",\n      \"11A\": \"\",\n      \"11B\": \"\",\n      \"11C\": \"\",\n      \"12A\": \"\",\n      \"12B\": \"\",\n      \"12C\": \"\",\n      \"13A\": \"\",\n      \"13B\": \"\",\n      \"13C\": \"\",\n      \"14A\": \"\",\n      \"14B\": \"\",\n      \"14C\": \"\",\n      \"15A\": \"\",\n      \"15B\": \"\",\n      \"15C\": \"\",\n      \"16A\": \"\",\n      \"16B\": \"\",\n      \"16C\": \"\",\n      \"17A\": \"\",\n      \"17B\": \"\",\n      \"17C\": \"\",\n      \"18A\": \"\",\n      \"18B\": \"\",\n      \"18C\": \"\",\n      \"19A\": \"\",\n      \"19B\": \"\",\n      \"19C\": \"\",\n      \"20A\": \"\",\n      \"20B\": \"\",\n      \"20C\": \"\",\n      \"21A\": \"\",\n      \"21B\": \"\",\n      \"21C\": \"\",\n      \"22A\": \"\",\n      \"22B\": \"\",\n      \"22C\": \"\",\n      \"23A\": \"\",\n      \"23B\": \"\",\n      \"23C\": \"\",\n      \"24A\": \"\",\n      \"24B\": \"\",\n      \"24C\": \"\",\n      \"25A\": \"\",\n      \"25B\": \"\",\n      \"25C\": \"\",\n      \"26A\": \"\",\n      \"26B\": \"\",\n      \"26C\": \"\",\n      \"27A\": \"\",\n      \"27B\": \"\",\n      \"27C\": \"\"\n    }\n  }\n}"

[2024-04-16T19:20:07.128Z] Error in interactWithFirestore
Error Details: request to https://www.googleapis.com/oauth2/v4/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
Stack Trace: Error: request to https://www.googleapis.com/oauth2/v4/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
    at Gaxios._request (C:\Users\theus\Yeaa\FCRdashboard\node_modules\gaxios\build\src\gaxios.js:148:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async GoogleToken.requestToken (C:\Users\theus\Yeaa\FCRdashboard\node_modules\gtoken\build\src\index.js:230:23)
    at async GoogleToken.getTokenAsync (C:\Users\theus\Yeaa\FCRdashboard\node_modules\gtoken\build\src\index.js:137:20)
    at async JWT.refreshTokenNoCache (C:\Users\theus\Yeaa\FCRdashboard\node_modules\google-auth-library\build\src\auth\jwtclient.js:165:23)
    at async JWT.refreshAccessTokenAsync (C:\Users\theus\Yeaa\FCRdashboard\node_modules\google-auth-library\build\src\auth\oauth2client.js:209:19)
    at async JWT.getAccessTokenAsync (C:\Users\theus\Yeaa\FCRdashboard\node_modules\google-auth-library\build\src\auth\oauth2client.js:238:23)
    at async interactWithFirestore (C:\Users\theus\Yeaa\FCRdashboard\server\models\firestoreApp.js:104:30)
    at async C:\Users\theus\Yeaa\FCRdashboard\server\routes.js:124:31
100

[2024-04-17T21:12:19.430Z] Error in writeInvoice
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "A": "012324Test",
  "B": "04/04/2024",
  "C": "Test Client",
  "D": "",
  "E": "",
  "creationDate": "2024-04-17T21:12:18.441Z",
  "invoiceDetails": {
    "1A": "",
    "1B": "",
    "1C": "",
    "2A": "LMIZ111222",
    "2B": "-m",
    "2C": "9.01",
    "3A": "(test)",
    "3B": "- Description 2",
    "3C": "411.19",
    "4A": "(anythingGoesHere)",
    "4B": "- MMMMMMMMMMMMM",
    "4C": "123.11",
    "5A": "",
    "5B": "- M again",
    "5C": "111111.11",
    "6A": "LMIZ111222",
    "6B": "-m",
    "6C": "9.01",
    "7A": "(test)",
    "7B": "- Description 2",
    "7C": "411.19",
    "8A": "(anythingGoesHere)",
    "8B": "",
    "8C": "NaN",
    "9A": "LMIZ111222",
    "9B": "-m",
    "9C": "9.01",
    "10A": "(test)",
    "10B": "- Description 2",
    "10C": "411.19",
    "11A": "(anythingGoesHere)",
    "11B": "",
    "11C": "NaN",
    "12C": "---------------------",
    "13B": "                                                                                                Total $ ",
    "13C": null
  }
}

[2024-04-17T21:13:55.333Z] Error in writeInvoice
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "A": "123",
  "B": "12/14/2024",
  "C": "Test Client",
  "D": "",
  "E": "",
  "creationDate": "2024-04-17T21:13:54.377Z",
  "invoiceDetails": {
    "1A": "",
    "1B": "",
    "1C": "",
    "2A": "LMIZ111222",
    "2B": "-m",
    "2C": "9.01",
    "3A": "(test)",
    "3B": "- Description 2",
    "3C": "411.19",
    "4A": "(anythingGoesHere)",
    "4B": "- MMMMMMMMMMMMM",
    "4C": "123.11",
    "5A": "",
    "5B": "- M again",
    "5C": "111111.11",
    "6C": "---------------------",
    "7B": "                                                                                                Total $ ",
    "7C": 111654.42
  }
}

[2024-04-17T21:15:18.395Z] Error in writeInvoice
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "A": "123",
  "B": "12/14/2024",
  "C": "Test Client",
  "D": "",
  "E": "",
  "creationDate": "2024-04-17T21:15:17.815Z",
  "invoiceDetails": {
    "1A": "",
    "1B": "",
    "1C": "",
    "2A": "LMIZ111222",
    "2B": "-m",
    "2C": "9.01",
    "3A": "(test)",
    "3B": "- Description 2",
    "3C": "411.19",
    "4A": "(anythingGoesHere)",
    "4B": "- MMMMMMMMMMMMM",
    "4C": "123.11",
    "5A": "",
    "5B": "- M again",
    "5C": "111111.11",
    "6A": "LMIZ111222",
    "6B": "-m",
    "6C": "9.01",
    "7A": "(test)",
    "7B": "- Description 2",
    "7C": "411.19",
    "8A": "(anythingGoesHere)",
    "8B": "",
    "8C": "NaN",
    "9A": "LMIZ111222",
    "9B": "-m",
    "9C": "9.01",
    "10A": "(test)",
    "10B": "- Description 2",
    "10C": "411.19",
    "11A": "(anythingGoesHere)",
    "11B": "",
    "11C": "NaN",
    "12C": "---------------------",
    "13B": "                                                                                                Total $ ",
    "13C": null
  }
}

[2024-04-18T01:35:32.198Z] Error in writeInvoice
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "A": "123",
  "B": "12/14/2024",
  "C": "Test Client",
  "D": "",
  "E": "",
  "creationDate": "2024-04-18T01:35:31.659Z",
  "invoiceDetails": {
    "1A": "",
    "1B": "",
    "1C": "",
    "2A": "LMIZ111222",
    "2B": "-m",
    "2C": 9.01,
    "3A": "(test)",
    "3B": "- Description 2",
    "3C": 411.19,
    "4A": "(anythingGoesHere)",
    "4B": "- MMMMMMMMMMMMM",
    "4C": 123.11,
    "5A": "",
    "5B": "- M again",
    "5C": 111111.11,
    "6C": "---------------------",
    "7B": "                                                                                                Total $ ",
    "7C": 111654.42
  }
}

[2024-04-18T01:36:43.557Z] Error in writeInvoice
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\FCRdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "A": "1212212",
  "B": "12/14/2024",
  "C": "Test Client",
  "D": "",
  "E": "",
  "creationDate": "2024-04-18T01:36:43.138Z",
  "invoiceDetails": {
    "1A": "",
    "1B": "",
    "1C": "",
    "2A": "LMIZ111222",
    "2B": "-m",
    "2C": 9.01,
    "3A": "(test)",
    "3B": "- Description 2",
    "3C": 411.19,
    "4A": "(anythingGoesHere)",
    "4B": "- MMMMMMMMMMMMM",
    "4C": 123.11,
    "5A": "",
    "5B": "- M again",
    "5C": 111111.11,
    "6C": "---------------------",
    "7B": "                                                                                                Total $ ",
    "7C": "111654.42"
  }
}

[2024-04-22T20:16:52.278Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "numberOfResults": "ALL",
  "fromAccount": "test",
  "startDate": "",
  "endDate": ""
}

[2024-04-22T20:18:02.412Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "numberOfResults": "ALL",
  "fromAccount": "5",
  "startDate": "",
  "endDate": ""
}

[2024-04-22T20:18:07.327Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "numberOfResults": "ALL",
  "fromAccount": "7",
  "startDate": "",
  "endDate": ""
}

[2024-04-22T20:18:15.559Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "numberOfResults": "ALL",
  "fromAccount": "test",
  "startDate": "",
  "endDate": ""
}

[2024-04-22T20:24:01.551Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "numberOfResults": "ALL",
  "fromAccount": "ALL",
  "startDate": "2024-12-13",
  "endDate": "2024-12-14"
}

[2024-04-23T21:03:23.817Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "numberOfResults": "ALL",
  "fromAccount": "ALL",
  "startDate": "2024-12-10",
  "endDate": "2024-12-15"
}

[2024-04-23T21:09:53.207Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "numberOfResults": "ALL",
  "fromAccount": "ALL",
  "startDate": "2024-12-10",
  "endDate": "2024-12-15"
}

[2024-04-23T21:12:07.259Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "numberOfResults": "ALL",
  "fromAccount": "ALL",
  "startDate": "2024-12-10",
  "endDate": "2024-12-15"
}

[2024-04-23T21:20:37.671Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "numberOfResults": "ALL",
  "fromAccount": "ALL",
  "startDate": "2024-12-10",
  "endDate": "2024-12-15"
}

[2024-04-23T21:22:21.285Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "numberOfResults": "ALL",
  "fromAccount": "ALL",
  "startDate": "2024-12-10",
  "endDate": "2024-12-15"
}

[2024-04-23T21:22:46.251Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "numberOfResults": "ALL",
  "fromAccount": "ALL",
  "startDate": "2024-12-10",
  "endDate": "2024-12-15"
}

[2024-04-23T21:36:40.583Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "numberOfResults": "ALL",
  "fromAccount": "ALL",
  "startDate": "2024-12-10",
  "endDate": "2024-12-15"
}

[2024-04-23T21:42:28.955Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{
  "numberOfResults": "ALL",
  "fromAccount": "ALL",
  "startDate": "2024-12-10",
  "endDate": "2024-12-15"
}

[2024-04-23T22:05:02.313Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\"numberOfResults\":\"ALL\",\"fromAccount\":\"ALL\",\"startDate\":\"2024-12-10\",\"endDate\":\"2024-12-15\"}"

[2024-04-23T22:11:31.333Z] Error in readAdvancedData
Error Details: firebase is not defined
Stack Trace: ReferenceError: firebase is not defined
    at formatDateForQuery (C:\Users\theus\Yeaa\fcrdashboard\server\models\firestoreApp.js:197:25)
    at interactWithFirestore (C:\Users\theus\Yeaa\fcrdashboard\server\models\firestoreApp.js:231:55)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async C:\Users\theus\Yeaa\fcrdashboard\server\routes.js:133:24
"{\"numberOfResults\":\"ALL\",\"fromAccount\":\"ALL\",\"startDate\":\"2024-12-10\",\"endDate\":\"2024-12-15\"}"

[2024-04-23T22:25:07.139Z] Error in readAdvancedData
Error Details: Request failed with status code 400
Response Data: [object Object]
Stack Trace: AxiosError: Request failed with status code 400
    at settle (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:1967:12)
    at Unzip.handleStreamEnd (C:\Users\theus\Yeaa\fcrdashboard\node_modules\axios\dist\node\axios.cjs:3062:11)
    at Unzip.emit (node:events:527:35)
    at endReadableNT (node:internal/streams/readable:1589:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
"{\"numberOfResults\":\"ALL\",\"fromAccount\":\"ALL\",\"startDate\":\"2024-12-10\",\"endDate\":\"2024-12-15\"}"

